playbook/app/pb_kits/playbook/pb_icon/_icon.jsx

```diff
@@ -83,17 +110,28 @@ const Icon = (props: IconProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
 
+  // Add a conditional here to show only the SVG if custom
   return (
     <>
-      <i
-          {...dataProps}
-          className={classes}
-          id={id}
-      />
-      <span
-          {...ariaProps}
-          hidden
-      />
+      <If condition={isCustomIcon}>
+        {
+          React.cloneElement(CUSTOM_ICONS[icon], {
+            ...dataProps,
+            className: classes,
+            id,
+          })
+        }
```

Comment:
The FA js won't convert SVGs, but all of the other classes can still be applied to the custom SVG just like our test. This is optimal bc now we can simply keep using the rest of what FA does style wise instead of re-inventing.

Reply:
Makes sense.